projects/m1/014-cell-phone-bill/js/index.js

```diff
@@ -0,0 +1,84 @@
+function askDataConsumed( dataType = '' ){
+
+    let message = 'How many minutes have you used up this month?';
+
+    if ( dataType === 'sms'){
+        message = 'And how many text messages?';
+    }

```

Comment:
I think it might be better to use a condition: 
if minutes .... 
otherwise if sms ....